packages/itwinui-react/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "compilerOptions": {
     "target": "es2020",
+    "module": "esnext",
```

Comment:
What is the reason for this change?

Reply:
This is for the `useInertPolyfill` function inside `Overlay.tsx`.